Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -547,7 +547,7 @@ bool DirectoryView::can_modify_current_selection()
 {
     auto selections = current_view().selection().indices();
     auto first_modifiable = selections.first_matching([&](auto& index) {
-        return !Core::System::access(node(index.parent()).full_path(), W_OK).is_error();
+        return Core::File::is_removable(node(index).full_path());
```

Comment:
This is now really confusing, the variable claims "modifyability", but the check tests for "removability". Not the same thing.

Reply:
I've used this to distinguish files you can remove/rename (files inside directory with write permission) from the modifiable files (these that can modify file contents – files with write permission). yeah, `is_removable()` isn't that a good name, but it's the best I could come up with. :/

actually wait maybe `can_delete_or_move()` would be a better choice?